.github/runTests.sh

```diff
@@ -0,0 +1,17 @@
+echo $PWD
+for file in $1
```

Comment:
What if there're multiple config files in one `parentdir`?

Reply:
add code to filter out unique changed directories